client/me/security-2fa-initial-setup/index.jsx

```diff
@@ -35,12 +54,39 @@ class Security2faInitialSetup extends Component {
 					) }
 				</p>
 
-				<FormButton
-					onClick={ ( event ) => {
-						gaRecordEvent( 'Me', 'Clicked On 2fa Get Started Button' );
-						this.props.onSuccess( event );
-					} }
-				>
+				<div>
+					<FormLabel className="security-2fa-initial-setup__label">
+						<FormRadio
+							name="auth_method"
+							value="app-based"
+							defaultChecked={ isAppSelected }
+							onChange={ this.setAuth.bind( this ) }
+						/>
+						<Gridicon icon="phone" />
+						<span>Set up using an app</span>
+						<p>
```

Comment:
Noting that a `<p />` inside a label isn't valid HTML. Generally speaking, block elements can't be nested inside inline elements. Perhaps we can move the `<p />` outside of the label?